.github/workflows/test.yml

```diff
@@ -13,13 +13,19 @@ jobs:
   build_and_test_ubuntu:
     runs-on: ubuntu-latest
     container:
-      image: rostooling/setup-ros-docker:ubuntu-jammy-latest
+      image: rostooling/setup-ros-docker:ubuntu-noble-latest
     steps:
     - uses: ros-tooling/action-ros-ci@v0.3
       with:
         package-name: libstatistics_collector
-        colcon-mixin-name: coverage-gcc
-        colcon-mixin-repository: https://raw.githubusercontent.com/colcon/colcon-mixin-repository/master/index.yaml
+        colcon-defaults: |
```

Comment:
I think this needs to be indented with 2 spaces: https://github.com/ros-tooling/libstatistics_collector/actions/runs/8882584262.